packages/core/bootstrap/README.md

```diff
@@ -1,28 +1,127 @@
+| :-------: | :-----------------------: | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :----------------: | :------------------------: |
+|           |      `CACHE_ENABLED`      |                                                                                                                                   Toggle caching.                                                                                                                                   |                    |          `false`           |
+|           |       `CACHE_TYPE`        |                                                                                                                          Which cache type should be used.                                                                                                                           | `local` or `redis` |          `local`           |
+|           |     `CACHE_KEY_GROUP`     |                                      Set to specific group ID to group the cached data, for this adapter, with other instances in the same group. Applicable only in remote cache scenarios, where multiple adapter instances share the cache.                                      |                    |    UUID of the adapter     |
+|           | `CACHE_KEY_IGNORED_PROPS` |                                                                                Keys to ignore while deriving the cache key, delimited by `,`. The key set will be added to the default ignored keys                                                                                 |                    | `['id', 'maxAge', 'meta']` |
+|           |      `CACHE_MAX_AGE`      | Maximum age in ms. Items are not pro-actively pruned out as they age, but if you try to get an item that is too old, it'll drop it and return undefined instead of giving it to you. If set to `0` the default will be used, and if set to `< 0` entries will not persist in cache. |                    |    `30000` (30 seconds)    |
+
+### Cache key
+
+The cache key of a stored request is derived by hashing the input object, using the SHA1 hash function, while by default ignoring keys `['id', 'maxAge']`. So for example these few requests will derive the same key:
+
+- `{"id": 1, "data": {"base":"LINK", "quote": "USD"}}`
+- `{"id": 2, "data": {"base":"LINK", "quote": "USD", "maxAge": 10000}}`
+- `{"id": 3, "data": {"base":"LINK", "quote": "USD"}}`
+
+The `maxAge` input argument can be used to set per item `maxAge` parameter. If not set, or set to `0`, the cache level `maxAge` option will be used. Every time the `maxAge` input argument changes, the item will be cached with the new `maxAge` parameter. To avoid hitting the cache for a specific item set `maxAge: -1` (any value `< 0`).
```

Comment:
I dont understand how someone would set the `maxAge` parameter, unless this is developer-only documentation.

Reply:
It could be added as a query string to the bridge URL.

Reply:
ah i see